yarn.lock

```diff
@@ -8309,6 +8309,15 @@ query-string@^4.1.0:
     object-assign "^4.1.0"
     strict-uri-encode "^1.0.0"
 
+query-string@^6.8.3:
```

Comment:
Add a .gitattributes to avoid this file expanding like this
https://github.com/bespoken/bst/blob/master/.gitattributes